web-components/src/components/molecules/CreditClassCardGrid/CreditClassCardGrid.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Card, CardMedia, SxProps } from '@mui/material';
+
+import { CreditPrice } from '../../../components/cards/ProjectCard/ProjectCard.CreditPrice';
+import { PurchaseInfo } from '../../../components/cards/ProjectCard/ProjectCard.types';
+import { Title } from '../../../components/typography';
+import { Theme } from '../../../theme/muiTheme';
+import { ButtonType } from '../../../types/shared/buttonType';
+import { LinkComponentType } from '../../../types/shared/linkComponentType';
+import { sxToArray } from '../../../utils/mui/sxToArray';
+import { truncateTwoLines } from '../../../utils/mui/truncate';
+import OutlinedButton from '../../buttons/OutlinedButton';
+
+export interface Props {
+  name: string;
+  imgSrc: string;
+  purchaseInfo?: PurchaseInfo;
+  button: ButtonType;
+  href?: string;
+  className?: string;
+  LinkComponent?: LinkComponentType;
+  sx?: SxProps<Theme>;
+}
+
+export const CreditClassCardGrid = ({
```

Comment:
```suggestion
export const CreditClassGridCard = ({
```
What about this naming instead? The current one made me think that this component was a grid of credit class cards while it's a credit class card that is meant to display within a grid, right?

Reply:
The empty state should simply hide this tab and default to the portfolio being open. In the future we will bring back the "create credit class" button in which case the private view would look like this: https://www.figma.com/file/seUiQM2SPt78KJiDL0Ea8m/Basic-Ecocredits-Ledger-View-%2F-Marketplace?type=design&node-id=5252%3A136395&mode=design&t=h6nRLiQ3ZfJTe1yu-1. But until we have the credit class flow fixed I guess we differ this and just hide the tab.

Reply:
`CreditClassGridCard ` looks good to me.